{"name": "test_add[6-5-30]", "status": "failed", "statusDetails": {"message": "AssertionError: Test Failed!!!\nassert 30 == 11", "trace": "self = <Work.test_params.TestParams object at 0x100fb5ef0>, a = 6, b = 5\nexpected = 30\n\n    @allure.description(\"加法计算\")\n    @pytest.mark.parametrize(\"a,b,expected\", [(5, 7, 9),\n                                              (4,4,8),\n                                              (9,9,18),\n                                              (40,40,1),\n                                              (6,5,30)])\n    def test_add(self,a,b,expected):\n        sum = add(a,b)\n>       assert expected == sum,\"Test Failed!!!\"\nE       AssertionError: Test Failed!!!\nE       assert 30 == 11\n\ntest_params.py:47: AssertionError"}, "description": "加法计算", "attachments": [{"name": "log", "source": "24c43c1c-e64d-47a3-93dd-b43658a2979b-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "966cb728-d996-42db-b764-f143d27b3981-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "a", "value": "6"}, {"name": "b", "value": "5"}, {"name": "expected", "value": "30"}], "start": 1572057907235, "stop": 1572057907236, "uuid": "11e2759a-6e00-4598-9548-6c8408f3b115", "historyId": "bf73f3d4a533c2b2864c4b0ecda83325", "testCaseId": "5e9e0a754422f4a1cda34a85c2742411", "fullName": "test_params.TestParams#test_add", "labels": [{"name": "suite", "value": "test_params"}, {"name": "subSuite", "value": "TestParams"}, {"name": "host", "value": "ParsondeMacBook-Pro.local"}, {"name": "thread", "value": "6382-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_params"}]}